src/lib/hooks/useBidData.ts

```diff
@@ -0,0 +1,21 @@
+//- React Imports
+import { useQuery } from 'react-query';
+
+//- Hooks Imports
+import { useZnsSdk } from './useZnsSdk';
+
+export const useBidData = (domainId: string) => {
+	// SDK
+	const sdk = useZnsSdk();
+
+	// Query
```

Comment:
nit: we can get rid of these comments, as the hook is simple enough without them. I generally add comments as the code becomes less self-explanatory, but it's pretty straight forward here.

Reply:
Okay yeah sounds good. Removed each case of these comments.